src/server/src/main/java/io/cassandrareaper/management/jmx/JmxCassandraManagementProxy.java

```diff
@@ -518,7 +518,7 @@ public void cancelAllRepairs() {
   }
 
   @Override
-  public Map<String, List<String>> listTablesByKeyspace() {
+  public Map<String, List<String>> listTablesByKeyspace() throws ReaperException {
```

Comment:
Question: I don't know about this. For example, this method is called in the [ClusterFacade](https://github.com/thelastpickle/cassandra-reaper/blob/cf727084454145c0c70e806f5aa57f117cef45c2/src/server/src/main/java/io/cassandrareaper/management/ClusterFacade.java#L399-L401), which returns the response directly. When called there, won't this exception being hit actually end up being an unhandled exception?

There is no try/catch block at that callsite.